lando/README.md

```diff
@@ -77,6 +80,13 @@ In order to sync from a staging or dev site, you will have to do the following:
 - `lando drush config-export` - export your local database settings
 - `lando drush config-import` - import new database settings to your local.
 
+Utilizing these commands with specific sites in your multisite setup looks like this: `lando drush @[]my-multisite] cr`.
+
+## Troubleshooting
+### Importing Configuration
+- If you run into issues importing new config files try running the command with the partial flag: `lando drush config-import --partial`.
```

Comment:
might want to add one other note here about if that partial flag doesn't work, then you should re-sync your whole database and then do `lando composer install` in case you are missing a dependency.